modules/optimization/test/tests/optimizationreporter/base/gold/optRep_fromCsv_out.json

```diff
@@ -1,173 +1,183 @@
 {
-  "reporters": {
```

Comment:
Why are these json files changing so much??? The names of the gradient vectors should be the only thing that's changing.